src/include/distributed/coordinator_protocol.h

```diff
@@ -194,7 +194,7 @@ extern uint64 GetNextShardId(void);
 extern uint64 GetNextPlacementId(void);
 extern Oid ResolveRelationId(text *relationName, bool missingOk);
 extern List * GetFullTableCreationCommands(Oid relationId, bool includeSequenceDefaults);
-extern List * GetPostLoadTableCreationCommands(Oid relationId, bool includeIndexes,
```

Comment:
I don't think you should remove this. In community we only give `true` for this parameter but in enterprise repo we also give `false`.

Reply:
yeah you're right, thank you